dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -111,14 +117,22 @@ internal class OkHttpUploader : Uploader {
         }
     }
 
+    private fun RequestBody.sizeInMb(): Double {
+        val megabyte = (1024 * 1024).toDouble()
+        return contentLength().toDouble() / megabyte
+    }
+
     // endregion
 
     companion object {
 
         internal val NETWORK_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(45)
         internal val MEDIA_TYPE_TXT = MediaType.parse("text/plain")
         internal val MEDIA_TYPE_JSON = MediaType.parse("application/json")
-
+        private const val MAX_MAP_FILE_SIZE_IN_MB = 50.0
```

Comment:
Why is this value a double and not a long ? Wouldn't it be easier to keep a 
`MAX_MAP_FILE_SIZE_IN_MB = 50L * 1024L * 1024L` and compare that to the actual size of the file?

Reply:
Yes....I really don't know why I did it the other way around :)